tests/file_contents_sorter_test.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from pre_commit_hooks.file_contents_sorter import FAIL
+from pre_commit_hooks.file_contents_sorter import main
+from pre_commit_hooks.file_contents_sorter import parse_commandline_input
+from pre_commit_hooks.file_contents_sorter import PASS
+
+
+def _n(*strs):
+    return b'\n'.join(strs) + b'\n'
```

Comment:
`_n` to me feels like "native string" -- I'd also just write the string literals directly -- generally [avoid logic in tests](https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html)